_posts/2021-08-31-Tez.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: project
+permalink: /:title/
+category: bigdata
+
+meta:
+  keywords: "Apache Tez"
+
+project:
+  title: "Apache Tez"
+  type: "Java"
+  url: "https://github.com/apache/tez"
+  project_official_url: "https://tez.apache.org/"
+  logo: "/assets/images/projects/libraries/ant/logo.png"
```

Comment:
```suggestion
  logo: "/assets/images/projects/libraries/tez/logo.png"
```

Reply:
Sorry for late.
I will submit a new pr to fix it.
Thank you.